htdocs/install/mysql/migration/21.0.0-22.0.0.sql

```diff
@@ -131,3 +131,7 @@ ALTER TABLE llx_categorie_supplier_invoice ADD INDEX idx_categorie_supplier_invo
 
 ALTER TABLE llx_categorie_supplier_invoice ADD CONSTRAINT fk_categorie_supplier_invoice_categorie_rowid FOREIGN KEY (fk_categorie) REFERENCES llx_categorie (rowid);
 ALTER TABLE llx_categorie_supplier_invoice ADD CONSTRAINT fk_categorie_supplier_invoice_fk_supplier_invoice_rowid FOREIGN KEY (fk_supplier_invoice) REFERENCES llx_facture_fourn (rowid);
+
+ALTER TABLE llx_propaldet ADD COLUMN subtotal_options text;
```

Comment:
Can you describe the goal of the field ?

There is already exisitng subtotal modules that works correctly (for example subtotal of ATM) using the existing fields. It should be a good idea to integrate them the same way of working first and then add features.